sass/_SimpleFilter.scss

```diff
@@ -9,6 +10,21 @@
   display: block;
   @include flex-wrap(nowrap);
   @include justify-content(flex-start);
+  padding: 0 0 0 15px;
+  @include mediaSelector(800) {
+    padding: 0 0 0 10px;
```

Comment:
Not using the shorter version of the padding property because someone would be unfamiliar with it would be a bad decision in my opinion. I don't think it's an obscure feature or that it's confusing in any way. If someone doesn't know about it they should look it up or ask.

For example, it would be pretty sad to not use array destructuring because someone might not be familiar with it.

Reply:
I don't see how adding 3 zeros makes for a shorter version. I am with using advanced syntax when the person is leveraging the benefits, but in this case we are not. I don't see why we would choose to look something up when we can be more explicit.

Reply:
I personally think that using the advanced syntax and having one less line is more concise, but I think this is a benign issue and I really don't mind which syntax is used in this case.

The point I wanted to get across is that using advanced syntax shouldn't be considered less explicit because the reader might not know about it :)

Reply:
Well, since there does not seem to be a consensus, I'll leave it like that.

One thing I prefer about single line shorthand is that it is not "possible" to separate the linked properties in the code.

By that I mean, if it was in 2 different lines, it would be possible to have something like this  : 
```
padding: 0;
padding-left: 15px;
```

become (over the years because someone need to modify the css, copy paste stuff around, refactor the sass modules, etc..) : 

```
padding: 0;
display: block;
position: relative;
margin: 12345px;
[... you get the idea ...]
padding-left: 15px;
```

And then the fact that those 2 lines are "closely" linked gets lost over time, and someone might add yet another override because it's harder to see that there are 2 rules about padding for that element.

When the rules are together on one line, it's at least much more harder to do that.

But I'd agree, this is all relatively minor.